magento/magento2
# [MFTF] Refactoring of AdminMassOrdersCancelCompleteAndClosedTest

### Description (*)
A new test has been created according to the best practices followed by MFTF.
The old "AdminMassOrdersCancelCompleteAndClosedTest" has been deprecated.

### Manual testing scenarios (*)
1. Create two orders
2. Invoice and ship the first order
3. Invoice and refund the second order
4. Go to the Orders Grid
5. Try to cancel both orders via mass update
6. Verify the result


### Resolved issues:
1. [x] resolves magento/magento2#31129: [MFTF] Refactoring of AdminMassOrdersCancelCompleteAndClosedTest
